app/models/merge_request.rb

```diff
@@ -31,4 +31,12 @@ def organisation_name_uniqueness
   def absorbing_organisation_name
     absorbing_organisation&.name || ""
   end
+
+  def requesting_user
+    User.find(requester_id) if requester_id
```

Comment:
Would it make sense to have requester as an association? 

Similar to absorbing organisation (`belongs_to :absorbing_organisation, class_name: "Organisation", optional: true` at the top of this file)

That way we should be able to access it as @merge_request.requester without having to create a method for it ourselves

Reply:
Sure, will try that